#!/usr/bin/env python3
# _*_ coding:utf-8 _*_

PACKAGE = "rm_detector"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("g_conf_thresh", double_t, 1, "g_conf_thresh", 0.5, 0.1, 1.0)
gen.add("g_nms_thresh",double_t, 2, "g_nms_thresh", 0.45, 0.1, 1.0)
gen.add("g_conf_thresh2", double_t, 3, "g_conf_thresh2", 0.5, 0.1, 1.0)
gen.add("g_nms_thresh2",double_t, 4, "g_nms_thresh2", 0.45, 0.1, 1.0)
gen.add("g_turn_on_image",bool_t,5,"g_turn_on_image",True)
gen.add("target_is_blue",bool_t,6,"target_is_blue",True)
gen.add("use_armor_detector", bool_t, 7,"use_armor_detector", True)
gen.add("use_class_model", bool_t, 7,"use_class_model", False)

exit(gen.generate(PACKAGE, PACKAGE, "dynamic"))